dotcom-rendering/src/components/Masthead/Titlepiece/EditionDropdown.tsx

```diff
@@ -51,6 +51,14 @@ const dropDownOverrides = css`
 	}
 `;
 
+const editionIdMap: Record<EditionId, string> = {
```

Comment:
Should this live in here? https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/edition.ts#L8

Reply:
Ah thanks Dom, I'll migrate 👍